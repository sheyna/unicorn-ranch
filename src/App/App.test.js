import App from './App';
import ReactTestRenderer from 'react-test-renderer';

describe('Home component', () => {
    it('should match snapshot', () => {
        const component = ReactTestRenderer.create(
            <App/>
        );
        expect(component.toJSON()).toMatchSnapshot();
    });
});
