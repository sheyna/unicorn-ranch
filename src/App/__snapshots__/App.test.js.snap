// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should match snapshot 1`] = `
<div
  className="App"
>
  <div>
    <header>
      <h1>
        Unicorns!
      </h1>
    </header>
    <main>
      <h2>
        Locations:
      </h2>
      <ul
        className="horseLocations"
      >
        <li>
          Barn
          <ul>
            <li>
              Purple
              <select
                id="Purple"
                name="Purple-location"
                onChange={[Function]}
                value="Barn"
              >
                <option
                  value="Barn"
                >
                  Barn
                </option>
                <option
                  value="Pasture"
                >
                  Pasture
                </option>
                <option
                  value="Trails"
                >
                  Trails
                </option>
              </select>
            </li>
            <li>
              Orange
              <select
                id="Orange"
                name="Orange-location"
                onChange={[Function]}
                value="Barn"
              >
                <option
                  value="Barn"
                >
                  Barn
                </option>
                <option
                  value="Pasture"
                >
                  Pasture
                </option>
                <option
                  value="Trails"
                >
                  Trails
                </option>
              </select>
            </li>
            <li>
              Green
              <select
                id="Green"
                name="Green-location"
                onChange={[Function]}
                value="Barn"
              >
                <option
                  value="Barn"
                >
                  Barn
                </option>
                <option
                  value="Pasture"
                >
                  Pasture
                </option>
                <option
                  value="Trails"
                >
                  Trails
                </option>
              </select>
            </li>
            <li>
              Pink
              <select
                id="Pink"
                name="Pink-location"
                onChange={[Function]}
                value="Barn"
              >
                <option
                  value="Barn"
                >
                  Barn
                </option>
                <option
                  value="Pasture"
                >
                  Pasture
                </option>
                <option
                  value="Trails"
                >
                  Trails
                </option>
              </select>
            </li>
          </ul>
        </li>
        <li>
          Pasture
          <ul />
        </li>
        <li>
          Trails
          <ul />
        </li>
      </ul>
    </main>
  </div>
</div>
`;
